spec/fixtures/api/crash.html

```diff
@@ -20,7 +20,7 @@
     })
 
     if (process.platform === 'win32') {
-      ipcRenderer.sendSync('crash-service-pid', crashReporter.crashServicePid)
```

Comment:
please revert this change, `crashServicePid` was fixed in https://github.com/electron/electron/pull/15206